README.md

```diff
@@ -64,6 +64,14 @@ By default, evil minions will respond as fast as possible, which might not be ap
  - `2.0` makes `evil-minion` react twice as slow as the original minion
  - `0.5` makes `evil-minion` react twice as fast as the original minion
 
+#### `--random-slowdown-factor` <True/False>
```

Comment:
How about making this variable rather than on/off?

What could be useful is expressing a random extra delay as a percentage of the original time.

BTW I hope we will not get into too much trouble with allowing out-of-order events this way. Otherwise we might need to make sure ordering is preserved despite adding randomness.